frontend/src/component/onboarding/flow/SdkExample.tsx

```diff
@@ -18,17 +18,42 @@ const StyledLink = styled(Link)({
     textDecoration: 'none',
 });
 
+const repositoryUrl =
+    'https://github.com/Unleash/unleash-sdk-examples/tree/main';
+const exampleDirectories: Record<SdkName, string> = {
```

Comment:
Could we get away without this mapping and name the directories the same as we have in app?

Reply:
Yes, if we deal with exceptions:
```
'.NET': 'Csharp',
'Node.js': 'NodeJS',
```